packages/documentation/package.json

```diff
@@ -39,6 +40,9 @@
     "gatsby"
   ],
   "license": "MIT",
+  "resolutions": {
+    "graphql": "14.1.1"
```

Comment:
gatsby blows up after installing this plugin with an error about multiple versions of graphql. i added this to force all of the dependencies to use this version.